client/src/styles.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const PhotosWrapper = styled.div`
+  display: flex;
+`;
+
+const PrimaryPhoto = styled.img`
+  max-width: 100%;
+  height: auto;
+`;
+
+export {
+  PhotosWrapper,
+  PrimaryPhoto
+}
```

Comment:
I appreciate the externalization of these styled-components. It reduces code in your main App file and makes the rest of the code easier to read.